Hearthstone Deck Tracker/Controls/Overlay/BobsBuddyPanel.xaml.cs

```diff
@@ -263,7 +264,8 @@ public async void ShowFullPanelAsync()
 				return;
 			//Await here so if they're mousing over a bunch of different heroes it doesn't stutter.
 			await Task.Delay(300);
-			MainPanel.Visibility = Visibility.Visible;
+			if(!OverlayWindow.MouseIsOverLeaderboardIcon)
```

Comment:
This should use `Core.Overlay` instead, and then `MouseIsOverLeaderboardIcon` does not need to be static